app/controllers/api/v2/hosts_controller.rb

```diff
@@ -353,7 +353,12 @@ def host_attributes(params, host = nil)
             interface_attributes(nic_attr, allow_nil_type: host.nil?)
           end
         end
-        params = host.apply_inherited_attributes(params) if host
+
+        if host
+          params = host.apply_inherited_attributes(params)
+          params = host.apply_interfaces_compute_attributes(params)
+        end
+
```

Comment:
Is this intentional? You assign to variable and then re-assign it once again, then return it?